packages/squiggle-lang/__tests__/Reducer/Reducer_MathJs/Reducer_MathJsParse_test.res

```diff
@@ -20,36 +18,34 @@ describe("MathJs parse", () => {
     test("(1+2)", () => expectParseToBe("(1+2)", "(add(1, 2))"))
   })
 
-  describe( "variables", () => {
+  describe("variables", () => {
     Skip.test("define", () => expectParseToBe("x = 1", "???"))
```

Comment:
I like that you wrote these tests for later, makes it clear what next steps will be

Reply:
RESOLVED